Exercise_3.py

```diff
@@ -1 +1,63 @@
 
+class Node:
+
+    def __init__(self,data):
+        self.data=data
+        self.next=None
+
+class LinkedList:
+    def __init__(self):
+        self.head=None
+
+
+    def push(self,data):
+        newnode=Node(data)
+        newnode.next=self.head
+        self.head=newnode
```

Comment:
can you give an explanation of your push method?

Reply:
every new node is added as self.head and all the new elements are added in front of the old add. In other words, the new element is pointed to the old head and then the new node becomes the new self.head

Reply:
Good
from now on try to give the explanation to your code using comments
this will help you alot